### ISSUE TITLE
Ability to translate objects

### ISSUE THREAD

**Issue by Gymnasiast** [OpenRCT2/OpenLoco#232]
This will be a long-term thing (obviously), but it would be nice if we could add object overrides like we did in OpenRCT2.

Especially since about everything in Locomotion is put into objects:
![locoscreenshot2](https://user-images.githubusercontent.com/1478678/45932578-8e5c5380-bf7e-11e8-96db-b4f5c478054c.png)

**Comment by Gymnasiast** [comment]
@Zcooger There already is. It's the `loco_original_id` variable in the language file. (0 = UK English, 1 = US English, 2 = French, 3= German, 4 = Spanish, 5 = Italian, 9 = Korean, 11 = Traditional Chinese).

**Comment by AaronVanGeffen** [comment]
@Zcooger Once we have implemented object loading, we will no longer be limited to Locomotion's memory model for string tables. (This will also mean we can store them in UTF-8 instead of CP1252.)

**Comment by Martinocom** [comment]
Is there any progress on this? The more I advance in the game the more (non-translated) messages I receive. I tried to understand the code and it seems to have all the translations embedded into the DAT object (as said [here](https://github.com/OpenLoco/OpenLoco/issues/232#issuecomment-423845373) but I read it later).

For example:

 > New coal seam found at [...] - Production up! 
 > <img width="500" alt="Image" src="https://github.com/user-attachments/assets/ee8fc7c2-db40-4ff7-8cd5-8cb5e3482ee4" />

This message does not appear in any of the translation files. It seems that (in this case) [MessageManager.cpp](https://github.com/OpenLoco/OpenLoco/blob/master/src/OpenLoco/src/MessageManager.cpp) loads the message directly.

A temp hack would be to read the english language (default) with available params and use it to do some pattern-matching and translations. Not ideal, but I'm open to suggestions. I have programming skills (not in C-like languages) but I can try to do something, if needed.

**Comment by AaronVanGeffen** [comment]
You're right, the templates for these messages are part of the industry objects. The current thinking is we'll make these translatable in a new object format that will extend the vanilla DAT format.

While we have thought about adding 'string overrides' for objects to the language files, the thinking was that it would be best to avoid such hacks so as to not make the codebase more complicated. However, the new object format has clearly not materialised yet.

Discussion on the new object format is ongoing, but on Discord. Please pitch in there, if you like.
